draft-ietf-quic-multipath.md

```diff
@@ -412,16 +414,16 @@ validated, the anti-amplification limit from {{Section 8 of QUIC-TRANSPORT}}
 applies.
 
 The server may receive packets for a yet unused Path ID that do not
-contain a path challenge. Such packets are valid if they can be properly decrypted
-and if they contain a valid connection ID.
+contain a PATH_CHALLENGE frame. Such packets are valid if they can be properly decrypted
+based on a valid connection ID.
```

Comment:
The new sentence is not correct: AEAD decryption is not based on connection ID, but based on a Path ID as part of nonce. The original text is correct on logic, but hard to read. I'd suggest the new text changes to:

```suggestion
and contain a valid Connection ID.
```

Reply:
The "based on" does not bother me, but the "given" version is maybe less ambiguous -- both of these versions works. I would not take the "and contain" as it would alter the original message of this sentence.